odoo/tools/float_utils.py

```diff
@@ -45,7 +45,7 @@ def float_round(value, precision_digits=None, precision_rounding=None, rounding_
        :param float value: the value to round
        :param int precision_digits: number of fractional digits to round to.
        :param float precision_rounding: decimal number representing the minimum
-           non-zero value at the desired precision (for example, 0.01 for a 
+           non-zero value at the desired precision (for example, 0.01 for a
```

Comment:
please don't do that

Reply:
We're talking about 7 lines of comment that are losing a space that shouldn't have been there in the first place. Frankly, I have other things to do than playing hide and seek with spaces for so little history "corruption". This is a pain in the ass, and we're going to feel it every single time we try touching this file.

Reply:
Or you could disable whatever is automatically removing trailing spaces and not getting unnecessary diff. Maybe the pain in the ass are these editors trying to be clever.

Reply:
... So that I could add trailing spaces everywhere else ? Or just activate/deactivate the option every time an file contain them ? That's the *very* definition of a pain in the ass. 
I don't like automatic stuff made by editors in general, but here it's minimal and useful.

Reply:
> ... So that I could add trailing spaces everywhere else ? Or just activate/deactivate the option every time a file contains them ? That's the _very_ definition of a pain in the ass.
> I don't like automatic stuff made by editors in general, but here it's minimal and useful.

Or configure your editor so that it only trims the lines you edited? Most editors should be capable of this.

Reply:
Hmmm... Actually, mine didn't, but I changed a few months ago (my eyes needed a true dark mode). I'll have a look.

Reply:
@xmo-odoo Sublime Text doesn't support it in its current version, but the next one will. I'll put the spaces back, alright :p